docs/sources/panels/expressions.md

```diff
@@ -101,13 +101,31 @@ While most functions exist in the own expression operations, the math operation
 
 abs returns the absolute value of its argument which can be a number or a series. For example `abs(-1)` or `abs($A)`.
 
+##### is_inf
+
+is_inf takes a number or a series and returns `1` for `Inf` values (negative or positive) and `0` for other values. For example `is_inf($A)`.
+
+> **Note:** If you need to specifically check for negative infinity for example, you can do a comparison like `$A == infn()`.
+
+##### is_nan
+
+is_nan takes a number or a series and returns `1` for `NaN` values and `0` for other values. For example `is_nan($A)`.The function exists because `NaN` is not equal to `NaN`.
```

Comment:
```suggestion
is_nan takes a number or a series and returns `1` for `NaN` values and `0` for other values. For example `is_nan($A)`. This function exists because `NaN` is not equal to `NaN`.
```